### PR TITLE
Add world of chunk support

### PR DESCRIPTION
**Pull request by TUCAOEVER**
Add support for "world of %chunk%" expression.

### Description
#4217 

---
**Target Minecraft Versions:** any
**Requirements:** 
**Related Issues:** #4215

### CODE REVIEW

**Comment by AyhamAl-Ali** [comment]
You should mention this issue (#4215) in your PR

**Comment by TUCAOEVER** [comment]
> You should mention this issue (#4215) in your PR

Thx for reminding.

File: src/main/java/ch/njol/skript/expressions/ExprWorld.java [hunk 1]
```diff
@@ -81,10 +83,11 @@ public World convert(final Object o) {
 						return ((PlayerTeleportEvent) e).getTo().getWorld();
 					else
 						return ((Entity) o).getWorld();
-				}
-				if (o instanceof Location)
+				} else if (o instanceof Location) {
 					return ((Location) o).getWorld();
-				assert false : o;
```
Comment (TPGamesNL) [inline src/main/java/ch/njol/skript/expressions/ExprWorld.java:87]: Keep the assertion (after the last if block)

File: src/main/java/ch/njol/skript/expressions/ExprWorld.java [hunk 2]
```diff
@@ -48,12 +49,13 @@
 @Description("The world the event occurred in.")
 @Examples({"world is \"world_nether\"",
 		"teleport the player to the world's spawn",
-		"set the weather in the player's world to rain"})
+		"set the weather in the player's world to rain",
+		"set {_world} to world of chunk at location of targeted entity"})
```
Comment (TPGamesNL) [inline src/main/java/ch/njol/skript/expressions/ExprWorld.java:53]: Not a good example (`world of targeted entity`), you also don't have to add an example. If you do want to, do something like `set {_world} to world of {_chunk}` or `set {_world} to world of event-chunk`

**Review by TPGamesNL — CHANGES_REQUESTED** [review]

**Review by TPGamesNL — APPROVED** [review]

**Comment by TPGamesNL** [comment]
@TUCAOEVER Next time, please don't delete your branch while the PR using it is still open \:)

**Pull request merged**